### PR TITLE
Help page for pull request

### PR DESCRIPTION
**Pull request by marcoemrich**
I created a detailed help page for the pull request. It's still WIP. Please take a look, if this makes sense. The most important point is to make people aware of Tras and the tests.s.

### CODE REVIEW

File: src/help_pull_request.ejs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  <div class="row">
+    <h5 class="col s12 m3 header">1. Go to <a href="https://github.com/jscraftcamp/website">our Github-Repo</a> and fork it.</h5>
+    <img  class="col s12 m9" src="/img/pr_screenshots/fork.png"/>
+  </div>
+  <div class="row">
+    <h5 class="col s12 m3 header">2. Clone <b>your fork</b>.</h5>
+    <img  class="col s12 m9" src="/img/pr_screenshots/clone.png"/>
+  </div>
+  <div class="row">
+    <h5 class="col s12 m12 header">3. Maker you changes, commit and push.</h5>
```
Comment (Narigo) [inline src/help_pull_request.ejs]: `Make your changes...`

File: src/help_pull_request.ejs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  </div>
+  <div class="row">
+    <h5 class="col s12 m3 header">4. Go to <a href="https://github.com/jscraftcamp/website">our Github-Repo</a> and create a <b>New pull request</b>. Click on <b>compare across forks</b></h5>
```
Comment (Narigo) [inline src/help_pull_request.ejs]: Missing `.` at the end?

File: src/help_pull_request.ejs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    <div class="col s12 m3">
+      <h5 class="header">7a2. See the Details</h5>
+      <p class="block">Here you will find the cause of the error. In the example screenshot on the right, the JSON was invalid therefore you get the somewhat strange "<i>before all hook</i>"-message. In other cases, you get more helpfull messages, like "<i>must contain a name</i>"</p>
+    </div>
+    <img  class="col s12 m9" src="/img/pr_screenshots/travis_details.png"/>
+  </div>
+
+  <div class="row">
+    <div class="col s12 m3">
+      <h5 class="header">7b. Everything 's green....</h5>
```
Comment (Narigo) [inline src/help_pull_request.ejs]: space

File: src/help_pull_request.ejs [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      <h5 class="header">7a1. If something goes wrong....</h5>
+      <p class="block">If Travis detects an error, click on <b>Details</b>, to get a detailed error log.</p>
+    </div>
+    <img  class="col s12 m9" src="/img/pr_screenshots/travis_fail.png"/>
+  </div>
+
+  <div class="row">
+    <div class="col s12 m3">
+      <h5 class="header">7a2. See the Details</h5>
+      <p class="block">Here you will find the cause of the error. In the example screenshot on the right, the JSON was invalid therefore you get the somewhat strange "<i>before all hook</i>"-message. In other cases, you get more helpfull messages, like "<i>must contain a name</i>"</p>
```
Comment (Narigo) [inline src/help_pull_request.ejs]: `helpfull` -> `helpful`
Missing `.` at the end?

**Comment by wolframkriesing** [comment]
That is a very nice page, is it also linked from the registration page, couldn't see it?

**Comment by marcoemrich** [comment]
Not yet. I just wanted to get your approval. before I add the link.

**Pull request merged**